### ISSUE TITLE
sys.excepthook not working as expected

### ISSUE THREAD

**Issue by Visherac** [20tab/UnrealEnginePython#274]
you gave an example with sys.excepthook working for PyQt. However it doesn't seem to be caught by other code that gets run and error hits.

Currently I am setting up a custom_handler and my "sys.excepthook = custom_handler" in ue_site.py that gets run on startup. But when I run code that hits an exception the custom_handler isn't hit. 

using python 2.7 in unreal engine 4.17 (using python 2.7 because that is what Maya uses and I have a built up shared code base)

**Comment by rdeioris** [comment]
Hi, is it a threaded version of the plugin ?

**Comment by Visherac** [comment]
Yes this is the threaded version. Sorry for the delay over the holidays.
